bsp/esp_bsp_generic/Kconfig

```diff
@@ -326,38 +326,87 @@ menu "Board Support Package"
     
 
     menu "LEDs"
-        config BSP_LEDS_NUM
-            int
-            prompt "Number of LEDs in BSP"
-            default 1
-            range 0 5
-            
         choice  
             prompt "LED type"
             default BSP_LED_TYPE_GPIO
             help
                 Select a LED type GPIO or Adressable RGB 
             config BSP_LED_TYPE_GPIO
                 bool "GPIO LED"
+            config BSP_LED_TYPE_RGB_CLASSIC
+                bool "Classic RGB LED"
             config BSP_LED_TYPE_RGB
                 bool "Adressable RGB LED"
         endchoice
+        
+        if BSP_LED_TYPE_GPIO
+            config ENV_MAX_LEDS
+                int
+                default 5
+        endif
+        if BSP_LED_TYPE_RGB_CLASSIC || BSP_LED_TYPE_RGB
```

Comment:
Should `ENV_MAX_LEDS` be very large for `BSP_LED_TYPE_RGB`? That means a very long led_strip?

With current settings the addressable RGB leds are limited to 1

Reply:
Because adressable RGB led is set as one, but you can control it by index in `led_indicator`.